site/components/pages/Pledge/PledgeForm/index.tsx

```diff
@@ -112,62 +113,86 @@ export const PledgeForm: FC<Props> = (props) => {
       )}
 
       <InputField
-        id="name"
+        inputProps={{
+          id: "name",
+          placeholder: t({
+            id: "pledges.form.input.name.placeholder",
+            message: "Name or company name",
+          }),
+          type: "text",
+          ...register("name"),
+        }}
         label={t({ id: "pledges.form.input.name.label", message: "Name" })}
-        placeholder={t({
-          id: "pledges.form.input.name.placeholder",
-          message: "Name or company name",
-        })}
-        type="text"
-        errors={formState.errors.name}
-        errorMessageMap={getPledgeFormErrorTranslations}
-        {...register("name")}
+        errorMessage={
+          !!formState.errors.name?.message &&
+          pledgeErrorTranslationsMap[
+            formState.errors.name.message as PledgeErrorId
+          ]
+        }
       />
 
       <InputField
-        id="profileImageUrl"
+        inputProps={{
+          id: "profileImageUrl",
+          placeholder: "https://",
+          type: "text",
+          ...register("profileImageUrl"),
+        }}
         label={t({
           id: "pledges.form.input.profileImageUrl.label",
           message: "Profile image url (optional)",
         })}
```

Comment:
boy seeing all these extra lines really makes me want to ditch ids 🤮
```
label={t`Profile image url (optional)`}
```

but I guess they are helpful for translators to find the string in our ui